tests/asgi/snapshots/snap_test_explorer.py

```diff
@@ -18,96 +35,69 @@
+  <head>
+    <title>Ariadne GraphQL</title>
+    <style>
+      body {
+        height: 100%;
+        margin: 0;
+        width: 100%;
+        overflow: hidden;
+      }
 
-<head>
-  <meta charset="utf-8" />
-  <title>Ariadne GraphQL</title>
-  <style>
-    body {
-      height: 100%;
-      margin: 0;
-      width: 100%;
-      overflow: hidden;
-    }
-
-    #graphiql {
-      height: 100vh;
-    }
-  </style>
-
-  <!--
-      This GraphiQL example depends on Promise and fetch, which are available in
-      modern browsers, but can be "polyfilled" for older browsers.
-      GraphiQL itself depends on React DOM.
-      If you do not want to rely on a CDN, you can host these files locally or
-      include them directly in your favored resource bundler.
-    -->
-  <script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
-  <script crossorigin src="https://unpkg.com/react-dom@17/umd/react-dom.development.js"></script>
-
-  <!--
-      These two files can be found in the npm module, however you may wish to
-      copy them directly into your environment, or perhaps include them in your
-      favored resource bundler.
-     -->
-  <link rel="stylesheet" href="https://unpkg.com/graphiql/graphiql.min.css" />
-  
-</head>
-
-<body>
-  <div id="graphiql">Loading Ariadne GraphQL...</div>
-  <script src="https://unpkg.com/graphiql/graphiql.min.js" type="application/javascript"></script>
-  
-  <script type="application/javascript">
-    var fetcher = GraphiQL.createFetcher({
-      url: 'https://swapi-graphql.netlify.app/.netlify/functions/index',
-    });
-    var defaultQuery = "# Welcome to GraphiQL\\
-#\\
-# GraphiQL is an in -browser tool for writing, validating, and\\
-# testing GraphQL queries.\\
-#\\
-# Type queries into this side of the screen, and you will see intelligent\\
-# typeaheads aware of the current GraphQL type schema and live syntax and\\
-# validation errors highlighted within the text.\\
-#\\
-# GraphQL queries typically start with a \\"{\\" character.Lines that start\\
-# with a # are ignored.\\
-#\\
-# An example GraphQL query might look like:\\
-#\\
-#     {\\
-#       field(arg: \\"value\\") {\\
-#         subField\\
-#\\
-        }\\
-#\\
-      }\\
-#\\
-# Keyboard shortcuts:\\
-#\\
-#   Prettify query: Shift - Ctrl - P(or press the prettify button)\\
-#\\
-#  Merge fragments: Shift - Ctrl - M(or press the merge button)\\
-#\\
-#        Run Query: Ctrl - Enter(or press the play button)\\
-#\\
-#    Auto Complete: Ctrl - Space(or just start typing)\\
-#";
+      #graphiql {
+        height: 100vh;
+      }
+    </style>
 
+    <link rel="stylesheet" href="https://unpkg.com/graphiql/graphiql.min.css" />
     
-    ReactDOM.render(
-      React.createElement(GraphiQL, {
-        fetcher: fetcher,
-        defaultEditorToolsVisibility: true,
-        query: query,
-      }),
-      document.getElementById('graphiql'),
-    );
+  </head>
+
+  <body>
+    <div id="graphiql">Loading...</div>
```

Comment:
Please take a note that we've included `{{ title }}` in the loading message.

Reply:
added {{ title }}